src/tweaksConfig.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * This configuration is used as an incubator for experimental features. Tweaks are not officially
+ * supported and are not guaranteed to be stable, i.e. their naming, functionality and API can
+ * change at any time within the tweaks or when being promoted to an official feature and moved
+ * into its final configuration namespace.
+ */
+export interface TweaksConfig {
+  /**
+   * The frequency in seconds onTimeChanged is called with TimeChangedEvents.
+   * Default is 1. Minimum is 0.025.
```

Comment:
I'll update the comment to reflect each platform's default value then. And I don't think we need to do much to handle them from the lib's perspective because the SDKs should do that already, and no default values are being defined from the library per se.

Reply:
Done 14608f24c81d242c23ff08db9495cdde38c9d253.